Library/Formula/snag.rb

```diff
@@ -16,46 +16,19 @@ class Snag < Formula
 
   depends_on "go" => :build
 
-  go_resource "gopkg.in/yaml.v2" do
```

Comment:
Yes, this new version of `snag` has all it's dependencies vendored. Nothing is fetched during compilation.